modules/tinyobjloader/2.0.0-rc1.bcr1/MODULE.bazel

```diff
@@ -0,0 +1,10 @@
+module(
+    name = "tinyobjloader",
+    version = "2.0.0-rc1.bcr1",
```

Comment:
should be  `version = "2.0.0-rc1.bcr.1",` (rename folder accordingly and all other occurances)